articles/client-platforms/react/index.yml

```diff
@@ -21,3 +21,5 @@ articles:
   - 02-custom-login
   - 03-session-handling
   - 04-user-profile
+  - 05-linking-accounts
```

Comment:
@flaviogranero 
The file has been updated from another PR, probably that's why this one has a conflict: https://github.com/auth0/docs/pull/1763

Please rebase.

Look into the file as it is now, there is already a `06-`: https://github.com/auth0/docs/tree/master/articles/client-platforms/react

Please coordinate with @alemata and update this PR accordingly.